client/dashboard/profile-wizard/style.scss

```diff
@@ -108,6 +108,41 @@
 		padding-right: 0;
 	}
 
+	.muriel-input-text {
+		&.empty {
+			.components-base-control__label {
+				display: none;
+			}
+
+			input {
+				top: 15px;
+			}
+		}
+
+		input[type='text']:focus {
+			box-shadow: none;
+			border: 0;
+		}
+	}
+
+	.muriel-input-text label,
+	.muriel-select label {
```

Comment:
The empty state label looks slightly off-center vertically.  Not sure if that's on the Muriel side or one of our styles affecting this:

```scss
.muriel-select.empty label {
    top: 15px;
}
```

Similarly the input fields with a value look a pixel or two off.

<img width="388" alt="Screen Shot 2019-05-28 at 1 59 08 PM" src="https://user-images.githubusercontent.com/10561050/58454540-5e8ace80-8151-11e9-9d6f-ff991f6a1455.png">